frontend/lib/src/components/widgets/DataFrame/hooks/useCustomTheme.ts

```diff
@@ -93,10 +110,15 @@ function useCustomTheme(): CustomThemeReturn {
   }, [theme])
 
   return {
-    theme: glideTheme,
+    glideTheme,
     tableBorderRadius: theme.radii.default,
-    tableBorderWidth: theme.sizes.borderWidth,
-    // Configure custom SVG icons used in the column header:
+    tableBorderWidth: 1,
+    defaultTableHeight: convertRemToPx("25rem"),
+    minColumnWidth: convertRemToPx("3.125rem"),
+    maxColumnWidth: convertRemToPx("62.5rem"),
+    maxColumnAutoWidth: convertRemToPx("31.25rem"),
+    defaultRowHeight: convertRemToPx("2.188rem"),
+    defaultHeaderHeight: convertRemToPx("2.188rem"),
```

Comment:
We might apply changes to all these variables to use nicer rem sizes in a follow-up PR.